Gemfile

```diff
@@ -2,7 +2,9 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gem 'faker', '~> 3.2.1'
-gem 'resource_api', git: 'https://github.com/performant-software/resource-api.git', tag: 'v0.5.3'
+gem 'jwt_auth', git: 'https://github.com/performant-software/jwt-auth.git', tag: 'v0.1.2'
+#gem 'resource_api', git: 'https://github.com/performant-software/resource-api.git', tag: 'v0.5.3'
+gem 'resource_api', path: '../resource-api'
```

Comment:
Is this configuration left over from local dev work?

Reply:
Ha, yes, reminding me to release a new version of the `resource_api` and update this Gemfile. Done.